pylutron/__init__.py

```diff
@@ -966,6 +966,7 @@ def handle_update(self, args):
 
 
 class OccupancyGroup(LutronEntity):
+  """Represents one or more occupancy/vacancy sensors grouped into an Area."""
```

Comment:
squash to the other occupancy changes.

Reply:
RE my other git complaints, I think I have ruined this branch to the point of having to start over :( Squashing gives me merge conflicts.